pkg/process/procutil/process_linux.go

```diff
@@ -1,3 +1,4 @@
+//go:build linux
```

Comment:
Is this added by your IDE? I'm not aware that we need this.

Reply:
This is actually added to `gofmt` by default. The syntax for compiler statements has changed, and `gofmt` adds this to help you transition. It supports both now, but eventually we will need to get rid of the old style